src/main/java/seedu/duke/Duke.java

```diff
@@ -7,23 +7,52 @@
 import seedu.duke.data.ShoppingList;
 import seedu.duke.parser.Parser;
 
+import java.io.File;
+import java.io.IOException;
 import java.util.Scanner;
+import java.util.logging.ConsoleHandler;
+import java.util.logging.FileHandler;
+import java.util.logging.Level;
+import java.util.logging.LogManager;
+import java.util.logging.Logger;
 
 public class Duke {
 
-    /**
-      * Main entry-point for the java.duke.Duke application.
-      *
-     */
     public static Budget myBudget = new Budget();
     private static ShoppingList items = new ShoppingList();
     private static Scanner in = new Scanner(System.in);
+    private static final Logger LOGGER = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
+    public static final String DEFAULT_FILEPATH = "Logger.log";
+    public static final String LOG_FILE_ERROR = "Logging to file unsuccessful";
 
-
+    /**
+     * Main entry-point for the java.duke.Duke application.
+     *
+     */
     public static void main(String[] args) {
+        Duke.setUpLogger();
         new Duke().run();
     }
 
+    private static void setUpLogger() {
+        LogManager.getLogManager().reset();
+        LOGGER.setLevel(Level.ALL);
+        ConsoleHandler console = new ConsoleHandler();
+        console.setLevel(Level.SEVERE);
+        LOGGER.addHandler(console);
+        File savedData = new File(DEFAULT_FILEPATH);
```

Comment:
Maybe can call this File object "logFile" or something along those lines? Cause "savedData" might be confusing later on when we add the save functionality for the shopping list

Reply:
Noted, thanks!